src/actinia_module_plugin/endpoints.py

```diff
@@ -76,15 +106,11 @@ def create_endpoints(flask_api):
     apidoc.add_resource(ListVirtualModules, "/modules")
     apidoc.add_resource(DescribeVirtualModule, "/modules/<module>")
 
-    apidoc.add_resource(
-        GdiAsyncEphemeralExportResource,
-        "/locations/<string:location_name>/processing_export",
-    )
+    # add deprecated location endpoints
+    create_project_endpoints(apidoc, projects_url_part="locations")
 
-    apidoc.add_resource(
-        GdiAsyncPersistentResource,
-        "/locations/<string:location_name>/mapsets/<string:mapset_name>/processing",
-    )
+    # add project endpoints
+    create_project_endpoints(apidoc, projects_url_part="projects")
```

Comment:
```suggestion
    # Endpoints for processing including deprecated locations endpoints
    create_project_endpoints()
    create_project_endpoints(projects_url_part="locations")
```
Suggestion is to align it to actinia-core.
What happens with `apidoc`? Does this need to be adjusted in actinia-core?

Reply:
We discussed it and I will make it more like actinia-core.